src/chttpd/src/chttpd_misc.erl

```diff
@@ -141,6 +144,38 @@ all_dbs_callback({error, Reason}, #vacc{resp=Resp0}=Acc) ->
     {ok, Resp1} = chttpd:send_delayed_error(Resp0, Reason),
     {ok, Acc#vacc{resp=Resp1}}.
 
+handle_dbs_info_req(#httpd{method='POST'}=Req) ->
+    chttpd:validate_ctype(Req, "application/json"),
+    Props = couch_httpd:json_body_obj(Req),
+    Keys = couch_mrview_util:get_view_keys(Props),
```

Comment:
I am not sure about it. On one side the `get_view_keys` does the validation we need. On the other side this has nothing to do with views.
Feel free to ignore.

Reply:
Above is just trying to get `keys` but we need to make sure that `keys` exists in current case. So added additional validation below.